api/metrics-api-jaxrs/pom.xml

```diff
@@ -93,10 +93,16 @@
     <!-- documentation -->
     <dependency>
       <groupId>com.wordnik</groupId>
-      <artifactId>swagger-annotations_2.9.1</artifactId>
-      <version>${swagger-annotations.version}</version>
+      <artifactId>swagger-annotations</artifactId>
+      <version>${swagger.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>com.wordnik</groupId>
+      <artifactId>swagger-core_2.10</artifactId>
+      <version>${swagger.version}</version>
+      <!--<scope>provided</scope>-->
```

Comment:
Ah, the commented line should have been removed. compile scope is fine now, since we don't use the interfaces on runtime currently.